lib/fxpay/settings.js

```diff
@@ -80,12 +83,18 @@
         exports[def] = defaultSettings[def];
       }
     }
-    for (var k in newSettings) {
-      if (typeof exports[k] === 'undefined') {
-        exports.log.error('configure() received an unknown setting:', k);
+    for (var param in newSettings) {
```

Comment:
Would be good to think about and document target support for this library - just so we can see what is and isn't working against those criteria. Even if that's a work in progress. For example we know that postMessaging from browser windows doesn't work in IE even in 11 [1]

[1] http://blogs.msdn.com/b/ieinternals/archive/2009/09/15/bugs-in-ie8-support-for-html5-postmessage-sessionstorage-and-localstorage.aspx